### ISSUE TITLE
Accept test set to test the final model

### ISSUE THREAD

**Issue by kojisekig** [LTR4L/ltr4l#46]
At present, our command line tool Main accepts training set and validation set and they are used during training, but we don't have the way of test the final model after the training finished.

**Comment by kojisekig** [comment]
This relates to #142
